recipes/perl-text-roman/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: perl-text-roman
+  version: "3.5"
+
+source:
+  url: https://cpan.metacpan.org/authors/id/S/SY/SYP/Text-Roman-3.5.tar.gz
+  md5: 1f6b09c0cc1f4425b565ff787a39fd83
+
+build:
+  noarch: generic
+  number: 2
+
+requirements:
+  build:
+    - make
+  host:
+    - perl
+    - perl-module-build
+  run:
+    - perl
+
```

Comment:
You need some tests:
```
test:
  commands:
    - ???
```

Reply:
Added the following test based on the other Perl modules in bioconda. 
```
test:
  imports:
    - Text::Roman
```